### ISSUE TITLE
VB -> C#: Converting RectangleF.op_Implicit 

### ISSUE THREAD

**Issue by AteCoder** [icsharpcode/CodeConverter#678]
### Input code
```vbnet
    Public Sub SetSize(ByVal clientRectangle As Rectangle)
        Dim area As RectangleF = RectangleF.op_Implicit(clientRectangle)
        area.Inflate(-Me._OuterGap, -Me._OuterGap)
        Me.PaneArea = area
    End Sub
```

### Erroneous output
```C#
        public void SetSize(Rectangle clientRectangle)
        {
            var area = RectangleF.op_Implicit(clientRectangle);
            area.Inflate(-OuterGap, -OuterGap);
            PaneArea = area;
        }
```

### Expected output
```C#
        public void SetSize(Rectangle clientRectangle)
        {
            RectangleF area = clientRectangle;
            area.Inflate(-OuterGap, -OuterGap);
            PaneArea = area;
        }
```

### Details
* Product in use: [codeconverter.icsharpcode.net](https://codeconverter.icsharpcode.net) / VS extension 
* Version in use: 8.1.8.0 artifact of 10/12
* Did you see it working in a previous version, which? N/A
* Any other relevant information to the issue, or your interest in contributing a fix. N.A

**Comment by GrahamTheCoder** [comment]
Interesting case, thanks. I asssume op_Implicit will fail too and any other operators that can be called by name like that. Would be good to check if there's a bigger list and fix them all together

**Comment by AteCoder** [comment]
https://docs.microsoft.com/en-us/dotnet/api/system.decimal.op_implicit?view=netcore-3.1
This one has an example. The RectangleF topic on this operator pointed to a non existing example. 

I am watching the change log to see when the items implemented in the 8.1.8.0 artifact I am using are released.

On a side note; I have fond memories of Cambridge. I stayed at the Emanuel College for. Conference on Human Timing years ago.  Had a chance to walk through the famous colleges, watch the painting of Henry the VIII, walk through the pastures. So different from LA.

**Comment by GrahamTheCoder** [comment]
Ah apologies, I suspect I missed the items out of the changelog because I wasn't sure they worked at the time of writing. All changes from the 8.1.8 artifact are included in all subsequent releases, so you should be able to uninstall that build and then install fresh from the marketplace.

Yeah Cambridge really is a lovely place to wander about, and to sit out in a beer garden drinking with friends!
